### ISSUE TITLE
Better color contrasting for DT

### ISSUE THREAD

**Issue by cpsievert** [rstudio/bslib#198]
During or after https://github.com/rstudio/DT/pull/852 is merged, we should make DT color contrasting work better with custom primary colors. For example, the color text on the pagination buttons here should be white, not black:

```r
library(shiny)

ui <- fluidPage(
  theme = bslib::bs_theme(bg = "black", fg = "white", primary = "#2754AE"),
  DT::datatable(mtcars, style = "bootstrap4")
)

server <- function(input, output) {}

shinyApp(ui, server)
```

**Comment by cpsievert** [comment]
This might be reason enough to pursue the https://github.com/rstudio/DT/pull/740 direction instead of rstudio/DT#852 (i.e., compile the datatable Sass at run-time against the theme)

**Comment by cpsievert** [comment]
Seems this is no longer much of an issue
